cmd/preflight/cli/run.go

```diff
@@ -161,3 +169,35 @@ func runPreflights(v *viper.Viper, arg string) error {
 
 	return showStdoutResults(v.GetString("format"), preflightSpec.Name, analyzeResults)
 }
+
+func parseTimeFlags(v *viper.Viper, progressChan chan interface{}, collectors []*troubleshootv1beta2.Collect) error {
+	var (
+		sinceTime time.Time
+		err       error
+	)
+	if v.GetString("since-time") != "" {
+		if v.GetString("since") != "" {
+			progressChan <- errors.Errorf("Only one of since-time / since may be used. The flag since-time will be used.")
```

Comment:
If `kubectl` errors on inclusion of both, I think we should do the same. If someone specifies both, it's a configuration error that they probably didn't mean to do. Is ignoring one of the arguments really the right thing?

Reply:
Makes sense, I changed it to return an error.